assets/js/blocks/cart-checkout/checkout/edit.js

```diff
@@ -146,21 +149,18 @@ const BlockSettings = ( { attributes, setAttributes } ) => {
 				) }
 			</PanelBody>
 			<PanelBody
-				title={ __( 'Content', 'woo-gutenberg-products-block' ) }
+				title={ __(
+					'Navigation options',
+					'woo-gutenberg-products-block'
+				) }
 			>
-				<p className="wc-block-checkout__controls-text">
-					{ __(
-						'Choose additional content to display.',
-						'woo-gutenberg-products-block'
-					) }
-				</p>
 				<ToggleControl
 					label={ __(
 						'Show links to policies',
 						'woo-gutenberg-products-block'
 					) }
 					help={ __(
-						'Shows a list of links to your "terms and conditions" and "privacy policy" pages.',
+						'Shows links to your "terms and conditions" and "privacy policy" pages.',
```

Comment:
Should the links be capitalized?
```suggestion
						'Shows links to your "Terms and conditions" and "Privacy policy" pages.',
```

Reply:
Maybe ... previously they weren't, and personally I try to avoid capitalisation if not useful (since these are in quotes too), so I'll skip this for now. Could be added later of course.